pages/lib/genTypeDefData.js

```diff
@@ -189,15 +188,15 @@ function DocVisitor(source) {
       ensureGroup(node);
 
       var propertyObj = {
-        line: getLineNum(node),
+        line: getLineNum(node)
```

Comment:
FYI, you can add `--trailing-comma=es5` or `--trailing-comma=all` to add them back